modelator/src/model/language/tla/mod.rs

```diff
@@ -168,23 +168,78 @@ impl Tla {
     }
 }
 
+/// Returns a list of operator names
+fn extract_operator_names(tla_file_contents: &str) -> Result<Vec<String>, Error> {
+    let cnt_operators = tla_file_contents.match_indices("==").count();
+    lazy_static! {
+        // Match '<identifier><whitespace>=='
+        static ref RE: Regex = Regex::new(r"([a-zA-Z0-9_]*)\s*==").unwrap();
```

Comment:
Maybe we can get away with this without `regex`. Look at how `extract_test_names` is done.

It would be good if we can keep the dependencies minimal.

Reply:
Ah but I lifted a common helper `extract operator names` and improve extract_test-names. The previous ad-hoc parsing logic in the implementation of `extract_test_names` would actually fail the current test. 

I know it is a pain to add more dependencies but I think regex is a common one which we would need at some point anyway. Probably at some point we should replace some existing parsing with regex IMO.